packages/hardhat-core/test/internal/hardhat-network/provider/modules/hardhat.ts

```diff
@@ -1,12 +1,15 @@
 import { assert } from "chai";
 import { bufferToHex } from "ethereumjs-util";
+import sinon from "sinon";
 
+import { numberToRpcQuantity } from "../../../../../internal/hardhat-network/provider/output";
```

Comment:
```suggestion
import { numberToRpcQuantity } from "../../../../../src/internal/hardhat-network/provider/output";
```
Please make sure that your imports always point to the files in `src` or `test` directory. This one points to a file that was built. Such imports are problematic because you cannot run tests in this file without building first.